---
title: " "
permalink: /PUBLICATIONS/
header:
  image: /assets/img/photos/header6.jpeg

toc: true
toc_sticky: true
toc_label: " "
toc_icon: "bars"
layout: single
# classes: wide
---

# Selected publications

[59] Bo Zhou#, Hongwen Huang#, Zhen Ding#, Kaiwen Luo, Yangshan Chen, Yingying Han, Wei Pang, Wanze Tang, Litong Chen, **Wenfei Jin***, Guixing Ma\*, Huiling Cao\*. Unveiling two distinct osteolineage cell populations linked to age‐related osteoporosis in adult mice through integrative single‐cell analyses. Cellular and Molecular Life Sciences. 2025. 82(1):106. <a href="https://link.springer.com/article/10.1007/s00018-025-05597-w" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[58] Weilong Yang\*, Jian-Kang Zhu\* and **Wenfei Jin***. A catalog of gene editing sites and genetic variations in editing sites in model organisms. BMC Genomics. 2024. 25(1):1153. <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-024-11073-9" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[57] Wenwen Cheng, Changhui Yin, Shiya Yu, Xi Chen, Ni Hong, **Wenfei Jin***. scMMO-atlas: a single cell multimodal omics atlas and portal for exploring fine cell heterogeneity and cell dynamics. Nucleic Acids Research. 2024. 53(D1):D1186-D1194. <a href="https://academic.oup.com/nar/article/53/D1/D1186/7771567?login=true" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[56] Wei Cao, Xuefei Wang, Kaiwen Luo, Yang Li, Jiahong Sun, Ruqing Fu, Qi Zhang, Ni Hong\*, Edwin Cheung\*, **Wenfei Jin***. Single cell analyses of cancer cells identified two regulatorily and functionally distinct categories in differentially expressed genes among tumor subclones. Heliyon. 2024. 10(6):e28071. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10958426/pdf/main.pdf" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[55] Guanqun Ren#, Yime Zhang#, Jiamin Liu#, Wenwen Cheng#, Di Wu, Mengwei Han, Yanyu Zeng, Xingyu Zhao, Luni Hu, Min Zeng, Rama Krishna Gurram, Xiaole Hu, Bo Zhou, Zhiyuan Hou, Jinfang Zhu, **Wenfei Jin***, Chao Zhong\*. Decreased GATA3 levels cause changed mouse cutaneous innate lymphoid cell fate, facilitating hair follicle recycling. Developmental Cell. 2024. 59:1-15. <a href="https://www.cell.com/developmental-cell/abstract/S1534-5807(24)00268-5?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1534580724002685%3Fshowall%3Dtrue" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[54] Chunliu Huang#, Xuefei Wang#, Yingzhao Wang#, Yongyi Feng, Xiumei Wang, Shan Chen, Peidong Yan, Jing Liao, Qi Zhang, Chengzhou Mao, Yang Li, Lixiang Wang, Xinyu Wang, Wei Yi, Weibin Cai, Shoudeng Chen, Ni Hong\*, Weiling He\*, Jun Chen\*, **Wenfei Jin***. Sirpα on tumor-associated myeloid cells restrains antitumor immunity in colorectal cancer independent of its interaction with CD47. Nature Cancer. 2024. 5(3):500-516. <a href="https://www.nature.com/articles/s43018-023-00691-z" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[53] Qiushi Xu#, Yunlong Zhang#, Wei Xu, Dong Liu, **Wenfei Jin**, Xi Chen\*, Ni Hong\*. The chromatin accessibility dynamics during cell fate specifications in zebrafish early embryogenesis. Nucleic Acids Research. 2024. 52(6):3106-3120. <a href="https://academic.oup.com/nar/article/52/6/3106/7607874?login=true" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[52] Tailin He#, Bo Zhou#, Guohuan Sun#, Qinnan Yan, Sixiong Lin, Guixing Ma, Qing Yao, Xiaohao Wu, Yiming Zhong, Donghao Gan, Shaochuan Huo, **Wenfei Jin**, Di Chen, Xiaochun Bai\*, Tao Cheng\*, Huiling Cao\* & Guozhi Xiao\*. The bone-liver interaction modulates immune and hematopoietic function through Pinch-Cxcl12-Mbl2 pathway. Cell Death & Differentiation. 2024. 31(1):90-105. <a href="https://www.nature.com/articles/s41418-023-01243-9" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[51] Xuefei Wang, Xinchao Wu, Ni Hong, **Wenfei Jin***. Progress in single-cell multimodal sequencing and multi-omics data integration. Biophysical Reviews. 2023. 16(1):13-28. <a href="https://link.springer.com/article/10.1007/s12551-023-01092-3" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[50] Jie Zhou#, Guanming Chen#, Jiuling Wang#, Bo Zhou, Xuemin Sun, Jinsong Wang, Shu Tang, Xiangju Xing, Xiaofei Hu, Yang Zhao, Yu Peng, Wenjiong Shi, Tingting Zhao, Yuzhang Wu, Hanbing Zhong, Ni Hong, Zhihua Ruan\*, Yi Zhang\*, **Wenfei Jin***. Anti-PD-1 therapy achieves favorable outcomes in HBV-positive non-liver cancer. Oncogenesis. 2023. 12(1):22. <a href="https://www.nature.com/articles/s41389-023-00468-0" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[49] Hui Li#, Hongyi Liu#, Yifei Liu#, Xuefei Wang, Shiya Yu, Hongwen Huang, Xiangru Shen, Qi Zhang, Ni Hong, **Wenfei Jin***. Exploring the dynamics and influencing factors of CD4 T cell activation using single-cell RNA-seq. iScience. 2023. 26(9):107588. <a href="https://www.cell.com/iscience/fulltext/S2589-0042(23)01665-6?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2589004223016656%3Fshowall%3Dtrue" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[48] Huiguang Yi\*, Yanling Lin, Qing Chang, **Wenfei Jin***. A fast and globally optimal solution for RNA-seq quantification. Briefings in Bioinformatics. 2023. 24(5):bbad298. <a href="https://academic.oup.com/bib/article/24/5/bbad298/7245794?login=true" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[47] Di Wu#, Zongxian Li#, Yime Zhang#, Yinlian Zhang#, Guanqun Ren#, Yanyu Zeng, Huiying Liu, Weiqiang Guan, Xingyu Zhao, Peng Li, Luni Hu, Zhiyuan Hou, Jingjing Gong, Jun Li, **Wenfei Jin**, Zeping Hu, Changtao Jiang, Houhua Li & Chao Zhong\*. Proline uptake promotes activation of lymphoid tissue inducer cells to maintain gut homeostasis. Nature Metabolism. 2023. 5(11):1953-1968. <a href="https://www.nature.com/articles/s42255-023-00908-6#auth-Chao-Zhong-Aff1" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[46] Yang Su#, Yifan Luo#, Peitao Zhang, Hong Lin, Weijie Pu, Hongyun Zhang, Huifang Wang, Yi Hao, Yihang Xiao, Xiaozhe Zhang, Xiayun Wei, Siyue Nie, Keren Zhang, Qiuyu Fu, Hao Chen, Niu Huang, Yan Ren, Mingxuan Wu, Billy Kwok Chong Chow, Xing Chen, **Wenfei Jin**, Fengchao Wang\*, Li Zhao\*, Feng Rao\*. Glucose-induced CRL4<sup>COP1</sup>-p53 axis amplifies glycometabolism to drive tumorigenesis. Molecular Cell. 2023. 83(13):2316-2331. <a href="https://www.cell.com/molecular-cell/fulltext/S1097-2765(23)00432-X?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS109727652300432X%3Fshowall%3Dtrue" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[45] Wei Xu#, Weilong Yang#, Yunlong Zhang, Yawen Chen, Ni Hong, Qian Zhang, Xuefei Wang, Yukun Hu, Kun Song, **Wenfei Jin***, Xi Chen\*. ISSAAC-seq enables sensitive and flexible multimodal profiling of chromatin accessibility and gene expression in single cells. Nature Methods. 2022. 19(10):1243-1249. <a href="https://www.nature.com/articles/s41592-022-01601-4" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[44] Junliang Wang#, Wei Chen#, Wenjun Yue, Wenhong Hou, Feng Rao, Hanbing Zhong, Yuanming Qi\*, Ni Hong\*, Ting Ni\*, **Wenfei Jin***. Comprehensive mapping of alternative polyadenylation site usage and its dynamics at single-cell resolution. Proceedings of the National Academy of Sciences of the United States of America. 2022. 119(49):e2113504119. <a href="https://www.pnas.org/doi/10.1073/pnas.2113504119" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[43] Xuefei Wang#, Xiangru Shen#, Shan Chen, Hongyi Liu, Ni Hong, Hanbing Zhong, Xi Chen, **Wenfei Jin***. Reinvestigation of Classic T Cell Subsets and Identification of Novel Cell Subpopulations by Single-Cell RNA Sequencing. The Journal of Immunology. 2022. 208(2):396-406. <a href="https://journals.aai.org/jimmunol/article/208/2/396/234729/Reinvestigation-of-Classic-T-Cell-Subsets-and" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[42] Haoda Wu#, Ruiqing Fu#, Yu-Hong Zhang, Zhiming Liu, Zhen-Hua Chen, Jingkai Xu, Yongji Tian\*, **Wenfei Jin**, Samuel Zheng Hao Wong\*, Qing-Feng Wu. Single-Cell RNA Sequencing Unravels Upregulation of Immune Cell Crosstalk in Relapsed Pediatric Ependymoma. Frontiers in Immunology. 2022. 13:903246. <a href="https://www.frontiersin.org/journals/immunology/articles/10.3389/fimmu.2022.903246/full" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[41] Md Alamin, Most Humaira Sultana, Xiangyang Lou, **Wenfei Jin***, Haiming Xu\*. Dissecting Complex Traits Using Omics Data: A Review on the Linear Mixed Models and Their Application in GWAS. Plants (Basel). 2022. 11(23):3277. <a href="https://www.mdpi.com/2223-7747/11/23/3277" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[40] Pengfei Qin#, Yakun Pang#, Wenhong Hou#, Ruiqing Fu#, Yingchi Zhang#, Xuefei Wang#, Guofeng Meng, Qifa Liu, Xiaofan Zhu, Ni Hong\*, Tao Cheng\*, **Wenfei Jin***. Integrated decoding hematopoiesis and leukemogenesis using single-cell sequencing and its medical implication. Cell Discovery. 2021. 7(1):2. <a href="https://www.nature.com/articles/s41421-020-00223-4" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[39] Huiguang Yi, Yanling Lin, Chengqi Lin, **Wenfei Jin***. Kssd: sequence dimensionality reduction by k-mer substring space sampling enables real-time large-scale datasets analysis. Genome Biology. 2021. 22(1):84. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02303-4" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[38] Wenhong Hou#, Li Duan#, Changyuan Huang#, Xingfu Li, Xiao Xu, Pengfei Qin, Ni Hong, Daping Wang\*, **Wenfei Jin***. Cross-Tissue Characterization of Heterogeneities of Mesenchymal Stem Cells and Their Differentiation Potentials. Frontiers in Cell and Developmental Biology. 2021. 9:781021. <a href="https://www.frontiersin.org/journals/cell-and-developmental-biology/articles/10.3389/fcell.2021.781021/full" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[37] Ruiqing Fu, Pengfei Qin, Xianghui Zou, Zhangli Hu, Ni Hong\*, Yun Wang\*, **Wenfei Jin***. A Comprehensive Characterization of Monoallelic Expression During Hematopoiesis and Leukemogenesis via Single-Cell RNA-Sequencing. Frontiers in Cell and Developmental Biology. 2021. 9:702897. <a href="https://www.frontiersin.org/journals/cell-and-developmental-biology/articles/10.3389/fcell.2021.702897/full" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[36] Zixiang Luo#, Chenyu Xu#, Zhen Zhang\*, **Wenfei Jin***. A topology-preserving dimensionality reduction method for single-cell RNA-seq data using graph autoencoder. Scientific Reports. 2021. 11(1):20028. <a href="https://www.nature.com/articles/s41598-021-99003-7" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[35] Nonthaphat Kent Wong#, Shumeng Luo#, Eudora Y D Chow, Fei Meng, Adenike Adesanya, Jiahong Sun, Herman M H Ma, **Wenfei Jin**, Wan-Chun Li, Shea Ping Yip\*, Chien-Ling Huang\*. The Tyrosine Kinase-Driven Networks of Novel Long Non-coding RNAs and Their Molecular Targets in Myeloproliferative Neoplasms. Frontiers in Cell and Developmental Biology. 2021. 9:643043. <a href="https://www.frontiersin.org/journals/cell-and-developmental-biology/articles/10.3389/fcell.2021.643043/full" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[34] Wei Xu, Yi Wen, Yingying Liang, Qiushi Xu, Xuefei Wang, **Wenfei Jin** & Xi Chen\*. A plate-based single-cell ATAC-seq workflow for fast and robust profiling of chromatin accessibility. Nature Protocols. 2021. 16(8):4084-4107. <a href="https://www.nature.com/articles/s41596-021-00583-5" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[33] Bo Zhou, **Wenfei Jin***. Visualization of Single Cell RNA-Seq Data Using t-SNE in R. Methods in Molecular Biology. 2020. 2117:159-167. <a href="https://link.springer.com/protocol/10.1007/978-1-0716-0301-7_8" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[32] Zhijun Han, Cui Kairong, Katarzyna Placek, Ni Hong, Chengqi Lin, Wei Chen, Keji Zhao\*, **Wenfei Jin***. Diploid genome architecture revealed by multi-omic data of hybrid mice. Genome Research. 2020. 30:1097-1106. <a href="https://genome.cshlp.org/content/30/8/1097.long" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[31] Wei Chen, Xuefei Wang, Gang Wei, Yin Huang, Yufang Shi, Dan Li, Shengnu Qiu, Bin Zhou, Junhong Cao, Meng Chen\*, Pengfei Qin\*, **Wenfei Jin***, Ting Ni\*. Single-Cell Transcriptome Analysis Reveals Six Subpopulations Reflecting Distinct Cellular Fates in Senescent Mouse Embryonic Fibroblasts. Frontiers in Genetics. 2020. 11:867. <a href="https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2020.00867/full" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[30] Xuekun Fu#, Bo Zhou#, Qinnan Yan#, Chu Tao, Lei Qin, Xiaohao Wu, Sixiong Lin, Sheng Chen, Yumei Lai, Xuenong Zou, Zengwu Shao, Meiqing Wang, Di Chen, **Wenfei Jin**, Youqiang Song, Huiling Cao\*, Ge Zhang\*, Guozhi Xiao\*. Kindlin-2 regulates skeletal homeostasis by modulating PTH1R in mice. Signal Transduction and Targeted Therapy. 2020. 5(1):297. <a href="https://www.nature.com/articles/s41392-020-00328-y" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[29] Jiji T Kurup, Zhijun Han, **Wenfei Jin**, Benjamin L Kidder\*. H4K20me3 methyltransferase SUV420H2 shapes the chromatin landscape of pluripotent embryonic stem cells. Development. 2020. 147(23):dev188516. <a href="https://journals.biologists.com/dev/article/147/23/dev188516/225957/H4K20me3-methyltransferase-SUV420H2-shapes-the" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[28] Li Wang#, Rong Qiu#, Zhaoyang Zhang, Zhijun Han, Chao Yao, Guojun Hou, Dai Dai, **Wenfei Jin**, Yuanjia Tang, Xiang Yu, Nan Shen\*. The MicroRNA miR-22 Represses Th17 Cell Pathogenicity by Targeting PTEN-Regulated Pathways. Immunohorizons. 2020. 4(6):308-318. <a href="https://journals.aai.org/immunohorizons/article/4/6/308/4148/The-MicroRNA-miR-22-Represses-Th17-Cell" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[27] Rong Qiu#, Xiang Yu#, Li Wang, Zhijun Han, Chao Yao, Yange Cui, Guojun Hou, Dai Dai, **Wenfei Jin**, Nan Shen\*. Inhibition of Glycolysis in Pathogenic TH17 Cells through Targeting a miR -21-Peli1-c-Rel Pathway Prevents Autoimmunity. The Journal of Immunology. 2020. 204(12):3160-3170. <a href="https://journals.aai.org/jimmunol/article/204/12/3160/107561/Inhibition-of-Glycolysis-in-Pathogenic-TH17-Cells" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[26] Huiling Cao#\*, Qinnan Yan#, Dong Wang#, Yumei Lai, Bo Zhou, Qi Zhang, **Wenfei Jin**, Simin Lin, Yiming Lei, Liting Ma, Yuxi Guo, Yishu Wang, Yilin Wang, Xiaochun Bai, Chuanju Liu, Jian Q Feng, Chuanyue Wu, Di Chen, Xu Cao, Guozhi Xiao\*. Focal adhesion protein Kindlin-2 regulates bone homeostasis in mice. Bone Research. 2020. 8:2. <a href="https://www.nature.com/articles/s41413-019-0073-8" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[25] Wei Wang, Gang Ren, Ni Hong\*, **Wenfei Jin***. Exploring the changing landscape of cell-to-cell variation after CTCF knockdown via single cell RNA-seq. BMC Genomics. 2019. 20(1):1015. <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6379-5" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[24] Binbin Lai#, Qingsong Tang#, **Wenfei Jin#**, Gangqing Hu#, Darawalee Wangsa, Kairong Cui, Benjamin Z Stanton, Gang Ren, Yi Ding, Ming Zhao, Shuai Liu, Jiuzhou Song, Thomas Ried, Keji Zhao\*. Trac-looping measures genome structure and chromatin accessibility. Nature Methods. 2018. 15(9):741-747. <a href="https://www.nature.com/articles/s41592-018-0107-y" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[23] Binbin Lai, Weiwu Gao, Kairong Cui, Wanli Xie, Qingsong Tang, **Wenfei Jin**, Gangqing Hu, Bing Ni, Keji Zhao\*. Publisher Correction: Principles of nucleosome organization revealed by single-cell micrococcal nuclease sequencing. Nature. 2018. 564(7735):E17. <a href="https://www.nature.com/articles/s41586-018-0690-1" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[22] Binbin Lai, Weiwu Gao, Kairong Cui, Wanli Xie, Qingsong Tang, **Wenfei Jin**, Gangqing Hu, Bing Ni, Keji Zhao\*. Principles of nucleosome organization revealed by single-cell micrococcal nuclease sequencing. Nature. 2018. 562(7726):281-285. <a href="https://www.nature.com/articles/s41586-018-0567-3" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[21] Gangqing Hu#\*, Kairong Cui#, Difeng Fang, Satoshi Hirose, Xun Wang, Darawalee Wangsa, **Wenfei Jin**, Thomas Ried, Pentao Liu, Jinfang Zhu, Ellen V. Rothenberg\*, Keji Zhao\*. Transformation of Accessible Chromatin and 3D Nucleome Underlies Lineage Commitment of Early T Cells. Immunity. 2018. 48(2):227-242. <a href="https://www.cell.com/immunity/fulltext/S1074-7613(18)30034-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1074761318300347%3Fshowall%3Dtrue" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[20] Gang Ren#, **Wenfei Jin#**, Kairong Cui#, Joseph Rodrigez, Gangqing Hu, Zhiying Zhang, Daniel R Larson, Keji Zhao\*. CTCF-Mediated Enhancer-Promoter Interaction Is a Critical Regulator of Cell-to-Cell Variation of Gene Expression. Molecular Cell. 2017. 67(6):1049-1058. <a href="https://www.cell.com/molecular-cell/fulltext/S1097-2765(17)30624-X?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS109727651730624X%3Fshowall%3Dtrue" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[19] Yi Zhang#, Wai Lim Ku#, Shuai Liu, Kairong Cui, **Wenfei Jin**, Qingsong Tang, William Lu, Bing Ni\*, Keji Zhao\*. Genome-wide identification of histone H2A and histone variant H2A.Z-interacting proteins by bPPI-seq. Cell Research. 2017. 27(10):1258-1274. <a href="https://www.nature.com/articles/cr2017112" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[18] Daniel C Kraushaar#, **Wenfei Jin#**, Alika Maunakea, Brian Abraham, Misook Ha, Keji Zhao\*. Erratum to: Genome-wide incorporation dynamics reveal distinct categories of turnover for the histone variant H3.3. Genome Biology. 2016. 17:21. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0886-3" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[17] **Wenfei Jin#**, Qingsong Tang#, Mimi Wan, Kairong Cui, Yi Zhang, Gang Ren, Bing Ni, Jeffrey Sklar, Teresa M Przytycka, Richard Childs, David Levens, Keji Zhao\*. Genome-wide detection of DNase I hypersensitive sites in single cells and FFPE tissue samples. Nature. 2015. 528(7580):142-6. <a href="https://www.nature.com/articles/nature15740" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[16] Hao Hu#\*, Xiang Liu#, **Wenfei Jin**, H Hilger Ropers, Thomas F Wienker. Evaluating information content of SNPs for sample-tagging in re-sequencing projects. Scientific Reports. 2015. 5:10247. <a href="https://www.nature.com/articles/srep10247" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[15] Haiyi Lou#, Shilin Li#, **Wenfei Jin**, Ruiqing Fu, Dongsheng Lu, Xinwei Pan, Huaigu Zhou, Yuan Ping, Li Jin, Shuhua Xu\*. Copy number variations and genetic admixtures in three Xinjiang ethnic minority groups. European Journal of Human Genetics. 2015. 23(4):536-42. <a href="https://www.nature.com/articles/ejhg2014134" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[14] **Wenfei Jin**, Ran Li, Ying Zhou, Shuhua Xu\*. Distribution of Ancestral Chromosomal Segments in Admixed Genomes and Its Implications for Inferring Population History and Admixture Mapping. European Journal of Human Genetics. 2014. 22: 930-937. <a href="https://www.nature.com/articles/ejhg2013265" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[13] Wan Isa Hatin, Ab Rajab Nur-Shafawati, Ali Etemad, **Wenfei Jin**, Pengfei Qin, Shuhua Xu, Li Jin, Soon-Guan Tan, Pornprot Limprasert, Merican Amir Feisal, Mohammed Rizman-Idid, Bin Alwi Zilfalil\*, HUGO Pan-Asian SNP Consortium. A genome wide pattern of population structure and admixture in peninsular Malaysia Malays. The HUGO journal. 2014. 8(1):5. <a href="https://thehugojournal.springeropen.com/articles/10.1186/s11568-014-0005-z" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[12] Shi Yan\*, Chuanchao Wang, Hongxiang Zheng, Wei Wang, Zhendong Qin, Lanhai Wei, Yi Wang, Xuedong Pan, Wenqing Fu, Yungang He, Lijun Xiong, **Wenfei Jin**, Shilin Li, Yu An, Hui Li, Li Jin\*. Y chromosomes of 40% Chinese descend from three Neolithic super-grandfathers. PLoS One. 2014. 9(8):e105691. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0105691" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[11] Pengfei Qin#, Zhiqiang Li#, **Wenfei Jin**, Dongsheng Lu, Haiyi Lou, Jiawei Shen, Li Jin\*, Yongyong Shi\*, Shuhua Xu\*. A panel of ancestry informative markers to estimate and correct potential effects of population stratification in Han Chinese. European Journal of Human Genetics. 2014. 22(2):248-53. <a href="https://www.nature.com/articles/ejhg2013111" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[10] Daniel C Kraushaar#, **Wenfei Jin#**, Alika Maunakea, Brian Abraham, Misook Ha, Keji Zhao\*. Genome-wide incorporation dynamics reveal distinct categories of turnover for the histone variant H3.3. Genome Biology. 2013. 14(10):R121. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-10-r121" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[9] Erli Wang#, **Wenfei Jin#**, Wenyuan Duan#, Bin Qiao, Shuna Sun, Guoying Huang, Kaihu Shi, Li Jin\*, Hongyan Wang\*. Association of two variants in SMAD7 with the risk of congenital heart disease in the Han Chinese population. PLoS One. 2013. 8(9):e72423. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0072423" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[8] **Wenfei Jin**, Sijia Wang, Haifeng Wang, Li Jin, Shuhua Xu\*. Exploring Population Admixture Dynamics Using Empirical and Simulated Genome-Wide Distribution of Ancestral Chromosomal Segments. American Journal of Human Genetics. 2012. 91:849-862. <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(12)00511-3?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0002929712005113%3Fshowall%3Dtrue" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[7] Shuhua Xu\*, **Wenfei Jin**. Population Genetics in the Genomic Era (Book Chapter in Population Genetics, Book edited by Dr. Stuart Raleigh, University of Northampton, UK), InTech, Rijeka, Croatia (ISBN 979-953-307-446-6). 2012. Page: 137-168. <a href="https://raw.githubusercontent.com/Sinh-Jinlab/Sinh-Jinlab.github.io/main/assets/pdf/InTech书籍章节031113.pdf" rel="nofollow noopener noreferrer" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[6] **Wenfei Jin**, Pengfei Qin, Haiyi Lou, Li Jin\*, Shuhua Xu\*. A Systematic Characterization of Genes Underlying both Complex and Mendelian Diseases. Human Molecular Genetics. 2012. 21(7):1611-1624. <a href="https://academic.oup.com/hmg/article/21/7/1611/2900796?login=true" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[5] **Wenfei Jin**, Shuhua Xu\*, Haifeng Wang, Yongguo Li, Yiping Shen, Bailin Wu, Li Jin\*. Genome-Wide Detection of Natural Selection in African Americans Pre-and Post-Admixture. Genome Research. 2012. 22 (3):519-527. <a href="https://genome.cshlp.org/content/22/3/519.long" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[4] Haiyi Lou#, Shilin Li#, Yajun Yang, Longli Kang, Xin Zhang, **Wenfei Jin**, Bailin Wu, Li Jin\*, Shuhua Xu\*. A map of copy number variations in Chinese populations. PLoS One. 2011. 6(11):e27341. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0027341" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[3] Shuhua Xu\*, Shilin Li, Yajun Yang, Jingze Tan, Haiyi Lou, **Wenfei Jin**, Ling Yang, Xuedong Pan, Jiucun Wang, Yiping Shen, Bailin Wu, Hongyan Wang, Li Jin\*. A genome-wide search for signals of high-altitude adaptation in Tibetans. Molecular Biology and Evolution. 2011. 28(2):1003-11. <a href="https://academic.oup.com/mbe/article/28/2/1003/1215735" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[2] Shuhua Xu#, Xianyong Yin#, Shilin Li, **Wenfei Jin**, Haiyi Lou, Ling Yang, Xiaohong Gong, Hongyan Wang, Yiping Shen, Xuedong Pan, Yungang He, Yajun Yang, Yi Wang, Wenqing Fu, Yu An, Jiucun Wang, Jingze Tan, Ji Qian, Xiaoli Chen, Xin Zhang, Yangfei Sun, Xuejun Zhang, Bailin Wu, Li Jin\*. Genomic dissection of population substructure of Han Chinese and its implication in association studies. The American Journal of Human Genetics. 2009. 85(6):762-74. <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(09)00470-4?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0002929709004704%3Fshowall%3Dtrue" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[1] Shuhua Xu, **Wenfei Jin**, Li Jin\*. Haplotype-sharing analysis showing Uyghurs are unlikely genetic donors. Molecular Biology and Evolution. 2009. 26(10):2197-206. <a href="https://academic.oup.com/mbe/article/26/10/2197/1103735" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

# All publications

## 2025

[1] Bo Zhou#, Hongwen Huang#, Zhen Ding#, Kaiwen Luo, Yangshan Chen, Yingying Han, Wei Pang, Wanze Tang, Litong Chen, **Wenfei Jin***, Guixing Ma\*, Huiling Cao\*. Unveiling two distinct osteolineage cell populations linked to age‐related osteoporosis in adult mice through integrative single‐cell analyses. Cellular and Molecular Life Sciences. 2025. 82(1):106. <a href="https://link.springer.com/article/10.1007/s00018-025-05597-w" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2024

[7] Weilong Yang\*, Jian-Kang Zhu\* and **Wenfei Jin***. A catalog of gene editing sites and genetic variations in editing sites in model organisms. BMC Genomics. 2024. 25(1):1153. <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-024-11073-9" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[6] Wenwen Cheng, Changhui Yin, Shiya Yu, Xi Chen, Ni Hong, **Wenfei Jin***. scMMO-atlas: a single cell multimodal omics atlas and portal for exploring fine cell heterogeneity and cell dynamics. Nucleic Acids Research. 2024. 53(D1):D1186-D1194. <a href="https://academic.oup.com/nar/article/53/D1/D1186/7771567?login=true" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[5] Wei Cao, Xuefei Wang, Kaiwen Luo, Yang Li, Jiahong Sun, Ruqing Fu, Qi Zhang, Ni Hong\*, Edwin Cheung\*, **Wenfei Jin***. Single cell analyses of cancer cells identified two regulatorily and functionally distinct categories in differentially expressed genes among tumor subclones. Heliyon. 2024. 10(6):e28071. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10958426/pdf/main.pdf" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[4] Guanqun Ren#, Yime Zhang#, Jiamin Liu#, Wenwen Cheng#, Di Wu, Mengwei Han, Yanyu Zeng, Xingyu Zhao, Luni Hu, Min Zeng, Rama Krishna Gurram, Xiaole Hu, Bo Zhou, Zhiyuan Hou, Jinfang Zhu, **Wenfei Jin***, Chao Zhong\*. Decreased GATA3 levels cause changed mouse cutaneous innate lymphoid cell fate, facilitating hair follicle recycling. Developmental Cell. 2024. 59:1-15. <a href="https://www.cell.com/developmental-cell/abstract/S1534-5807(24)00268-5?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1534580724002685%3Fshowall%3Dtrue" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[3] Chunliu Huang#, Xuefei Wang#, Yingzhao Wang#, Yongyi Feng, Xiumei Wang, Shan Chen, Peidong Yan, Jing Liao, Qi Zhang, Chengzhou Mao, Yang Li, Lixiang Wang, Xinyu Wang, Wei Yi, Weibin Cai, Shoudeng Chen, Ni Hong\*, Weiling He\*, Jun Chen\*, **Wenfei Jin***. Sirpα on tumor-associated myeloid cells restrains antitumor immunity in colorectal cancer independent of its interaction with CD47. Nature Cancer. 2024. 5(3):500-516. <a href="https://www.nature.com/articles/s43018-023-00691-z" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[2] Qiushi Xu#, Yunlong Zhang#, Wei Xu, Dong Liu, **Wenfei Jin**, Xi Chen\*, Ni Hong\*. The chromatin accessibility dynamics during cell fate specifications in zebrafish early embryogenesis. Nucleic Acids Research. 2024. 52(6):3106-3120. <a href="https://academic.oup.com/nar/article/52/6/3106/7607874?login=true" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[1] Tailin He#, Bo Zhou#, Guohuan Sun#, Qinnan Yan, Sixiong Lin, Guixing Ma, Qing Yao, Xiaohao Wu, Yiming Zhong, Donghao Gan, Shaochuan Huo, **Wenfei Jin**, Di Chen, Xiaochun Bai\*, Tao Cheng\*, Huiling Cao\* & Guozhi Xiao\*. The bone-liver interaction modulates immune and hematopoietic function through Pinch-Cxcl12-Mbl2 pathway. Cell Death & Differentiation. 2024. 31(1):90-105. <a href="https://www.nature.com/articles/s41418-023-01243-9" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2023

[6] Xuefei Wang, Xinchao Wu, Ni Hong, **Wenfei Jin***. Progress in single-cell multimodal sequencing and multi-omics data integration. Biophysical Reviews. 2023. 16(1):13-28. <a href="https://link.springer.com/article/10.1007/s12551-023-01092-3" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a> 

[5] Jie Zhou#, Guanming Chen#, Jiuling Wang#, Bo Zhou, Xuemin Sun, Jinsong Wang, Shu Tang, Xiangju Xing, Xiaofei Hu, Yang Zhao, Yu Peng, Wenjiong Shi, Tingting Zhao, Yuzhang Wu, Hanbing Zhong, Ni Hong, Zhihua Ruan\*, Yi Zhang\*, **Wenfei Jin***. Anti-PD-1 therapy achieves favorable outcomes in HBV-positive non-liver cancer. Oncogenesis. 2023. 12(1):22. <a href="https://www.nature.com/articles/s41389-023-00468-0" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[4] Hui Li#, Hongyi Liu#, Yifei Liu#, Xuefei Wang, Shiya Yu, Hongwen Huang, Xiangru Shen, Qi Zhang, Ni Hong, **Wenfei Jin***. Exploring the dynamics and influencing factors of CD4 T cell activation using single-cell RNA-seq. iScience. 2023. 26(9):107588. <a href="https://www.cell.com/iscience/fulltext/S2589-0042(23)01665-6?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2589004223016656%3Fshowall%3Dtrue" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a> 

[3] Huiguang Yi\*, Yanling Lin, Qing Chang, **Wenfei Jin***. A fast and globally optimal solution for RNA-seq quantification. Briefings in Bioinformatics. 2023. 24(5):bbad298. <a href="https://academic.oup.com/bib/article/24/5/bbad298/7245794?login=true" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[2] Di Wu#, Zongxian Li#, Yime Zhang#, Yinlian Zhang#, Guanqun Ren#, Yanyu Zeng, Huiying Liu, Weiqiang Guan, Xingyu Zhao, Peng Li, Luni Hu, Zhiyuan Hou, Jingjing Gong, Jun Li, **Wenfei Jin**, Zeping Hu, Changtao Jiang, Houhua Li & Chao Zhong\*. Proline uptake promotes activation of lymphoid tissue inducer cells to maintain gut homeostasis. Nature Metabolism. 2023. 5(11):1953-1968. <a href="https://www.nature.com/articles/s42255-023-00908-6#auth-Chao-Zhong-Aff1" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[1] Yang Su#, Yifan Luo#, Peitao Zhang, Hong Lin, Weijie Pu, Hongyun Zhang, Huifang Wang, Yi Hao, Yihang Xiao, Xiaozhe Zhang, Xiayun Wei, Siyue Nie, Keren Zhang, Qiuyu Fu, Hao Chen, Niu Huang, Yan Ren, Mingxuan Wu, Billy Kwok Chong Chow, Xing Chen, **Wenfei Jin**, Fengchao Wang\*, Li Zhao\*, Feng Rao\*. Glucose-induced CRL4<sup>COP1</sup>-p53 axis amplifies glycometabolism to drive tumorigenesis. Molecular Cell. 2023. 83(13):2316-2331. <a href="https://www.cell.com/molecular-cell/fulltext/S1097-2765(23)00432-X?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS109727652300432X%3Fshowall%3Dtrue" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2022

[5] Wei Xu#, Weilong Yang#, Yunlong Zhang, Yawen Chen, Ni Hong, Qian Zhang, Xuefei Wang, Yukun Hu, Kun Song, **Wenfei Jin***, Xi Chen\*. ISSAAC-seq enables sensitive and flexible multimodal profiling of chromatin accessibility and gene expression in single cells. Nature Methods. 2022. 19(10):1243-1249. <a href="https://www.nature.com/articles/s41592-022-01601-4" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[4] Junliang Wang#, Wei Chen#, Wenjun Yue, Wenhong Hou, Feng Rao, Hanbing Zhong, Yuanming Qi\*, Ni Hong\*, Ting Ni\*, **Wenfei Jin***. Comprehensive mapping of alternative polyadenylation site usage and its dynamics at single-cell resolution. Proceedings of the National Academy of Sciences of the United States of America. 2022. 119(49):e2113504119. <a href="https://www.pnas.org/doi/10.1073/pnas.2113504119" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[3] Xuefei Wang#, Xiangru Shen#, Shan Chen, Hongyi Liu, Ni Hong, Hanbing Zhong, Xi Chen, **Wenfei Jin***. Reinvestigation of Classic T Cell Subsets and Identification of Novel Cell Subpopulations by Single-Cell RNA Sequencing. The Journal of Immunology. 2022. 208(2):396-406. <a href="https://journals.aai.org/jimmunol/article/208/2/396/234729/Reinvestigation-of-Classic-T-Cell-Subsets-and" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[2] Haoda Wu#, Ruiqing Fu#, Yu-Hong Zhang, Zhiming Liu, Zhen-Hua Chen, Jingkai Xu, Yongji Tian\*, **Wenfei Jin**, Samuel Zheng Hao Wong\*, Qing-Feng Wu. Single-Cell RNA Sequencing Unravels Upregulation of Immune Cell Crosstalk in Relapsed Pediatric Ependymoma. Frontiers in Immunology. 2022. 13:903246. <a href="https://www.frontiersin.org/journals/immunology/articles/10.3389/fimmu.2022.903246/full" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[1] Md Alamin, Most Humaira Sultana, Xiangyang Lou, **Wenfei Jin***, Haiming Xu\*. Dissecting Complex Traits Using Omics Data: A Review on the Linear Mixed Models and Their Application in GWAS. Plants (Basel). 2022. 11(23):3277. <a href="https://www.mdpi.com/2223-7747/11/23/3277" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2021

[7] Pengfei Qin#, Yakun Pang#, Wenhong Hou#, Ruiqing Fu#, Yingchi Zhang#, Xuefei Wang#, Guofeng Meng, Qifa Liu, Xiaofan Zhu, Ni Hong\*, Tao Cheng\*, **Wenfei Jin***. Integrated decoding hematopoiesis and leukemogenesis using single-cell sequencing and its medical implication. Cell Discovery. 2021. 7(1):2. <a href="https://www.nature.com/articles/s41421-020-00223-4" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[6] Huiguang Yi, Yanling Lin, Chengqi Lin, **Wenfei Jin***. Kssd: sequence dimensionality reduction by k-mer substring space sampling enables real-time large-scale datasets analysis. Genome Biology. 2021. 22(1):84. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02303-4" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[5] Wenhong Hou#, Li Duan#, Changyuan Huang#, Xingfu Li, Xiao Xu, Pengfei Qin, Ni Hong, Daping Wang\*, **Wenfei Jin***. Cross-Tissue Characterization of Heterogeneities of Mesenchymal Stem Cells and Their Differentiation Potentials. Frontiers in Cell and Developmental Biology. 2021. 9:781021. <a href="https://www.frontiersin.org/journals/cell-and-developmental-biology/articles/10.3389/fcell.2021.781021/full" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[4] Ruiqing Fu, Pengfei Qin, Xianghui Zou, Zhangli Hu, Ni Hong\*, Yun Wang\*, **Wenfei Jin***. A Comprehensive Characterization of Monoallelic Expression During Hematopoiesis and Leukemogenesis via Single-Cell RNA-Sequencing. Frontiers in Cell and Developmental Biology. 2021. 9:702897. <a href="https://www.frontiersin.org/journals/cell-and-developmental-biology/articles/10.3389/fcell.2021.702897/full" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[3] Zixiang Luo#, Chenyu Xu#, Zhen Zhang\*, **Wenfei Jin***. A topology-preserving dimensionality reduction method for single-cell RNA-seq data using graph autoencoder. Scientific Reports. 2021. 11(1):20028. <a href="https://www.nature.com/articles/s41598-021-99003-7" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[2] Nonthaphat Kent Wong#, Shumeng Luo#, Eudora Y D Chow, Fei Meng, Adenike Adesanya, Jiahong Sun, Herman M H Ma, **Wenfei Jin**, Wan-Chun Li, Shea Ping Yip\*, Chien-Ling Huang\*. The Tyrosine Kinase-Driven Networks of Novel Long Non-coding RNAs and Their Molecular Targets in Myeloproliferative Neoplasms. Frontiers in Cell and Developmental Biology. 2021. 9:643043. <a href="https://www.frontiersin.org/journals/cell-and-developmental-biology/articles/10.3389/fcell.2021.643043/full" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[1] Wei Xu, Yi Wen, Yingying Liang, Qiushi Xu, Xuefei Wang, **Wenfei Jin** & Xi Chen\*. A plate-based single-cell ATAC-seq workflow for fast and robust profiling of chromatin accessibility. Nature Protocols. 2021. 16(8):4084-4107. <a href="https://www.nature.com/articles/s41596-021-00583-5" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2020

[8] Bo Zhou, **Wenfei Jin***. Visualization of Single Cell RNA-Seq Data Using t-SNE in R. Methods in Molecular Biology. 2020. 2117:159-167. <a href="https://link.springer.com/protocol/10.1007/978-1-0716-0301-7_8" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[7] Zhijun Han, Cui Kairong, Katarzyna Placek, Ni Hong, Chengqi Lin, Wei Chen, Keji Zhao\*, **Wenfei Jin***. 2020. Diploid genome architecture revealed by multi-omic data of hybrid mice. Genome Research. 30:1097-1106. <a href="https://genome.cshlp.org/content/30/8/1097.long" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[6] Wei Chen, Xuefei Wang, Gang Wei, Yin Huang, Yufang Shi, Dan Li, Shengnu Qiu, Bin Zhou, Junhong Cao, Meng Chen\*, Pengfei Qin\*, **Wenfei Jin***, Ting Ni\*. Single-Cell Transcriptome Analysis Reveals Six Subpopulations Reflecting Distinct Cellular Fates in Senescent Mouse Embryonic Fibroblasts. Frontiers in Genetics. 2020. 11:867. <a href="https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2020.00867/full" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[5] Xuekun Fu#, Bo Zhou#, Qinnan Yan#, Chu Tao, Lei Qin, Xiaohao Wu, Sixiong Lin, Sheng Chen, Yumei Lai, Xuenong Zou, Zengwu Shao, Meiqing Wang, Di Chen, **Wenfei Jin**, Youqiang Song, Huiling Cao\*, Ge Zhang\*, Guozhi Xiao\*. Kindlin-2 regulates skeletal homeostasis by modulating PTH1R in mice. Signal Transduction and Targeted Therapy. 2020. 5(1):297. <a href="https://www.nature.com/articles/s41392-020-00328-y" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[4] Jiji T Kurup, Zhijun Han, **Wenfei Jin**, Benjamin L Kidder\*. H4K20me3 methyltransferase SUV420H2 shapes the chromatin landscape of pluripotent embryonic stem cells. Development. 2020. 147(23):dev188516. <a href="https://journals.biologists.com/dev/article/147/23/dev188516/225957/H4K20me3-methyltransferase-SUV420H2-shapes-the" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[3] Li Wang#, Rong Qiu#, Zhaoyang Zhang, Zhijun Han, Chao Yao, Guojun Hou, Dai Dai, **Wenfei Jin**, Yuanjia Tang, Xiang Yu, Nan Shen\*. The MicroRNA miR-22 Represses Th17 Cell Pathogenicity by Targeting PTEN-Regulated Pathways. Immunohorizons. 2020. 4(6):308-318. <a href="https://journals.aai.org/immunohorizons/article/4/6/308/4148/The-MicroRNA-miR-22-Represses-Th17-Cell" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[2] Rong Qiu#, Xiang Yu#, Li Wang, Zhijun Han, Chao Yao, Yange Cui, Guojun Hou, Dai Dai, **Wenfei Jin**, Nan Shen\*. Inhibition of Glycolysis in Pathogenic TH17 Cells through Targeting a miR -21-Peli1-c-Rel Pathway Prevents Autoimmunity. The Journal of Immunology. 2020. 204(12):3160-3170. <a href="https://journals.aai.org/jimmunol/article/204/12/3160/107561/Inhibition-of-Glycolysis-in-Pathogenic-TH17-Cells" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[1] Huiling Cao#\*, Qinnan Yan#, Dong Wang#, Yumei Lai, Bo Zhou, Qi Zhang, **Wenfei Jin**, Simin Lin, Yiming Lei, Liting Ma, Yuxi Guo, Yishu Wang, Yilin Wang, Xiaochun Bai, Chuanju Liu, Jian Q Feng, Chuanyue Wu, Di Chen, Xu Cao, Guozhi Xiao\*. Focal adhesion protein Kindlin-2 regulates bone homeostasis in mice. Bone Research. 2020. 8:2. <a href="https://www.nature.com/articles/s41413-019-0073-8" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2019

[1] Wei Wang, Gang Ren, Ni Hong\*, **Wenfei Jin***. Exploring the changing landscape of cell-to-cell variation after CTCF knockdown via single cell RNA-seq. BMC Genomics. 2019. 20(1):1015. <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6379-5" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2018

[4] Binbin Lai#, Qingsong Tang#, **Wenfei Jin#**, Gangqing Hu#, Darawalee Wangsa, Kairong Cui, Benjamin Z Stanton, Gang Ren, Yi Ding, Ming Zhao, Shuai Liu, Jiuzhou Song, Thomas Ried, Keji Zhao\*. Trac-looping measures genome structure and chromatin accessibility. Nature Methods. 2018. 15(9):741-747. <a href="https://www.nature.com/articles/s41592-018-0107-y" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[3] Binbin Lai, Weiwu Gao, Kairong Cui, Wanli Xie, Qingsong Tang, **Wenfei Jin**, Gangqing Hu, Bing Ni, Keji Zhao\*. Publisher Correction: Principles of nucleosome organization revealed by single-cell micrococcal nuclease sequencing. Nature. 2018. 564(7735):E17. <a href="https://www.nature.com/articles/s41586-018-0690-1" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[2] Binbin Lai, Weiwu Gao, Kairong Cui, Wanli Xie, Qingsong Tang, **Wenfei Jin**, Gangqing Hu, Bing Ni, Keji Zhao\*. Principles of nucleosome organization revealed by single-cell micrococcal nuclease sequencing. Nature. 2018. 562(7726):281-285. <a href="https://www.nature.com/articles/s41586-018-0567-3" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[1] Gangqing Hu#\*, Kairong Cui#, Difeng Fang, Satoshi Hirose, Xun Wang, Darawalee Wangsa, **Wenfei Jin**, Thomas Ried, Pentao Liu, Jinfang Zhu, Ellen V. Rothenberg\*, Keji Zhao\*. Transformation of Accessible Chromatin and 3D Nucleome Underlies Lineage Commitment of Early T Cells. Immunity. 2018. 48(2):227-242. <a href="https://www.cell.com/immunity/fulltext/S1074-7613(18)30034-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1074761318300347%3Fshowall%3Dtrue" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2017

[2] Gang Ren#, **Wenfei Jin#**, Kairong Cui#, Joseph Rodrigez, Gangqing Hu, Zhiying Zhang, Daniel R Larson, Keji Zhao\*. CTCF-Mediated Enhancer-Promoter Interaction Is a Critical Regulator of Cell-to-Cell Variation of Gene Expression. Molecular Cell. 2017. 67(6):1049-1058. <a href="https://www.cell.com/molecular-cell/fulltext/S1097-2765(17)30624-X?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS109727651730624X%3Fshowall%3Dtrue" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[1] Yi Zhang#, Wai Lim Ku#, Shuai Liu, Kairong Cui, **Wenfei Jin**, Qingsong Tang, William Lu, Bing Ni\*, Keji Zhao\*. Genome-wide identification of histone H2A and histone variant H2A.Z-interacting proteins by bPPI-seq. Cell Research. 2017. 27(10):1258-1274. <a href="https://www.nature.com/articles/cr2017112" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2016

[1] Daniel C Kraushaar#, **Wenfei Jin#**, Alika Maunakea, Brian Abraham, Misook Ha, Keji Zhao\*. Erratum to: Genome-wide incorporation dynamics reveal distinct categories of turnover for the histone variant H3.3. Genome Biology. 2016. 17:21. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0886-3" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2015

[3] **Wenfei Jin#**, Qingsong Tang#, Mimi Wan, Kairong Cui, Yi Zhang, Gang Ren, Bing Ni, Jeffrey Sklar, Teresa M Przytycka, Richard Childs, David Levens, Keji Zhao\*. Genome-wide detection of DNase I hypersensitive sites in single cells and FFPE tissue samples. Nature. 2015. 528(7580):142-6. <a href="https://www.nature.com/articles/nature15740" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[2] Hao Hu#\*, Xiang Liu#, **Wenfei Jin**, H Hilger Ropers, Thomas F Wienker. Evaluating information content of SNPs for sample-tagging in re-sequencing projects. Scientific Reports. 2015. 5:10247. <a href="https://www.nature.com/articles/srep10247" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[1] Haiyi Lou#, Shilin Li#, **Wenfei Jin**, Ruiqing Fu, Dongsheng Lu, Xinwei Pan, Huaigu Zhou, Yuan Ping, Li Jin, Shuhua Xu\*. Copy number variations and genetic admixtures in three Xinjiang ethnic minority groups. European Journal of Human Genetics. 2015. 23(4):536-42. <a href="https://www.nature.com/articles/ejhg2014134" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2014

[4] **Wenfei Jin**, Ran Li, Ying Zhou, Shuhua Xu\*. Distribution of Ancestral Chromosomal Segments in Admixed Genomes and Its Implications for Inferring Population History and Admixture Mapping. European Journal of Human Genetics. 2014. 22: 930-937. <a href="https://www.nature.com/articles/ejhg2013265" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[3] Wan Isa Hatin, Ab Rajab Nur-Shafawati, Ali Etemad, **Wenfei Jin**, Pengfei Qin, Shuhua Xu, Li Jin, Soon-Guan Tan, Pornprot Limprasert, Merican Amir Feisal, Mohammed Rizman-Idid, Bin Alwi Zilfalil\*, HUGO Pan-Asian SNP Consortium. A genome wide pattern of population structure and admixture in peninsular Malaysia Malays. The HUGO journal. 2014. 8(1):5. <a href="https://thehugojournal.springeropen.com/articles/10.1186/s11568-014-0005-z" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[2] Shi Yan\*, Chuanchao Wang, Hongxiang Zheng, Wei Wang, Zhendong Qin, Lanhai Wei, Yi Wang, Xuedong Pan, Wenqing Fu, Yungang He, Lijun Xiong, **Wenfei Jin**, Shilin Li, Yu An, Hui Li, Li Jin\*. Y chromosomes of 40% Chinese descend from three Neolithic super-grandfathers. PLoS One. 2014. 9(8):e105691. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0105691" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[1] Pengfei Qin#, Zhiqiang Li#, **Wenfei Jin**, Dongsheng Lu, Haiyi Lou, Jiawei Shen, Li Jin\*, Yongyong Shi\*, Shuhua Xu\*. A panel of ancestry informative markers to estimate and correct potential effects of population stratification in Han Chinese. European Journal of Human Genetics. 2014. 22(2):248-53. <a href="https://www.nature.com/articles/ejhg2013111" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2013

[2] Daniel C Kraushaar#, **Wenfei Jin#**, Alika Maunakea, Brian Abraham, Misook Ha, Keji Zhao\*. Genome-wide incorporation dynamics reveal distinct categories of turnover for the histone variant H3.3. Genome Biology. 2013. 14(10):R121. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-10-r121" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[1] Erli Wang#, **Wenfei Jin#**, Wenyuan Duan#, Bin Qiao, Shuna Sun, Guoying Huang, Kaihu Shi, Li Jin\*, Hongyan Wang\*. Association of two variants in SMAD7 with the risk of congenital heart disease in the Han Chinese population. PLoS One. 2013. 8(9):e72423. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0072423" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2012

[4] **Wenfei Jin**, Sijia Wang, Haifeng Wang, Li Jin, Shuhua Xu\*. Exploring Population Admixture Dynamics Using Empirical and Simulated Genome-Wide Distribution of Ancestral Chromosomal Segments. American Journal of Human Genetics. 2012. 91:849-862. <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(12)00511-3?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0002929712005113%3Fshowall%3Dtrue" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[3] Shuhua Xu\*, **Wenfei Jin**. Population Genetics in the Genomic Era (Book Chapter in Population Genetics, Book edited by Dr. Stuart Raleigh, University of Northampton, UK), InTech, Rijeka, Croatia (ISBN 979-953-307-446-6). 2012. Page: 137-168. <a href="https://raw.githubusercontent.com/Sinh-Jinlab/Sinh-Jinlab.github.io/main/assets/pdf/InTech书籍章节031113.pdf" rel="nofollow noopener noreferrer" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[2] **Wenfei Jin**, Pengfei Qin, Haiyi Lou, Li Jin\*, Shuhua Xu\*. A Systematic Characterization of Genes Underlying both Complex and Mendelian Diseases. Human Molecular Genetics. 2012. 21(7):1611-1624. <a href="https://academic.oup.com/hmg/article/21/7/1611/2900796?login=true" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[1] **Wenfei Jin**, Shuhua Xu\*, Haifeng Wang, Yongguo Li, Yiping Shen, Bailin Wu, Li Jin\*. Genome-Wide Detection of Natural Selection in African Americans Pre-and Post-Admixture. Genome Research. 2012. 22 (3):519-527. <a href="https://genome.cshlp.org/content/22/3/519.long" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2011

[2] Haiyi Lou#, Shilin Li#, Yajun Yang, Longli Kang, Xin Zhang, **Wenfei Jin**, Bailin Wu, Li Jin\*, Shuhua Xu\*. A map of copy number variations in Chinese populations. PLoS One. 2011. 6(11):e27341. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0027341" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[1] Shuhua Xu\*, Shilin Li, Yajun Yang, Jingze Tan, Haiyi Lou, **Wenfei Jin**, Ling Yang, Xuedong Pan, Jiucun Wang, Yiping Shen, Bailin Wu, Hongyan Wang, Li Jin\*. A genome-wide search for signals of high-altitude adaptation in Tibetans. Molecular Biology and Evolution. 2011. 28(2):1003-11. <a href="https://academic.oup.com/mbe/article/28/2/1003/1215735" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

## 2009

[2] Shuhua Xu#, Xianyong Yin#, Shilin Li, **Wenfei Jin**, Haiyi Lou, Ling Yang, Xiaohong Gong, Hongyan Wang, Yiping Shen, Xuedong Pan, Yungang He, Yajun Yang, Yi Wang, Wenqing Fu, Yu An, Jiucun Wang, Jingze Tan, Ji Qian, Xiaoli Chen, Xin Zhang, Yangfei Sun, Xuejun Zhang, Bailin Wu, Li Jin\*. Genomic dissection of population substructure of Han Chinese and its implication in association studies. The American Journal of Human Genetics. 2009. 85(6):762-74. <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(09)00470-4?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0002929709004704%3Fshowall%3Dtrue" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

[1] Shuhua Xu, **Wenfei Jin**, Li Jin\*. Haplotype-sharing analysis showing Uyghurs are unlikely genetic donors. Molecular Biology and Evolution. 2009. 26(10):2197-206. <a href="https://academic.oup.com/mbe/article/26/10/2197/1103735" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>


